tests/test_name.py

```diff
@@ -420,6 +421,18 @@ def test_parsed_name_initials():
     assert expected == parsed_name.first_initials_list
 
 
+def test_parsed_wrong_names_and_not_fail():
+    names = [('Proffesor.M.', 'Proffesor.M.'),
+             ('ˇ Sirˇ', u'Sirˇ, ˇ.'),
```

Comment:
why this has ``u`` and the other don't? consistency :)

Reply:
I added `u` because it outputs utf string, but I can change everything to utf strings